### PR TITLE
[RFC] more sniffs for better spacing

### PR DESCRIPTION
**Pull request by michz**
Please comment.

### CODE REVIEW

File: easy-coding-standard.neon [hunk 1]
```diff
@@ -109,6 +109,14 @@ checkers:
     - SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff
     - SlevomatCodingStandard\Sniffs\Commenting\RequireOneLinePropertyDocCommentSniff
 
+    SlevomatCodingStandard\Sniffs\Commenting\DocCommentSpacingSniff:
+        linesCountBeforeFirstContent: 0
+        linesCountBetweenDifferentAnnotationsTypes: 1
+
+    PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\MemberVarSpacingSniff:
+        spacing: 1
+        spacingBeforeFirst: 0
```
Comment (michz) [inline easy-coding-standard.neon:118]: This forbids:

```
class ChooseAdditionalSizeController extends MultiStepBaseController
{

    /** @var RepositoryInterface */
    private $productAttributesRepository;
    /** @var ProductRepositoryInterface */
    private $productRepository;
```

And enforces:

```
class ChooseAdditionalSizeController extends MultiStepBaseController
{
    /** @var RepositoryInterface */
    private $productAttributesRepository;

    /** @var ProductRepositoryInterface */
    private $productRepository;
```

**Review by mattagohni — APPROVED** [review]

File: easy-coding-standard.neon [hunk 2]
```diff
@@ -109,6 +109,14 @@ checkers:
     - SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff
     - SlevomatCodingStandard\Sniffs\Commenting\RequireOneLinePropertyDocCommentSniff
 
+    SlevomatCodingStandard\Sniffs\Commenting\DocCommentSpacingSniff:
+        linesCountBeforeFirstContent: 0
+        linesCountBetweenDifferentAnnotationsTypes: 1
```
Comment (michz) [inline easy-coding-standard.neon:114]: This forbids:

```
    /**
     *
     * Sort array by order defined in sortOrder array
     * @param string[] $input
     * @param string[] $sortOrder
     * @return string[]
     */
```

And enforces:

```
    /**
     * Sort array by order defined in sortOrder array
     *
     * @param string[] $input
     * @param string[] $sortOrder
     *
     * @return string[]
     */
```

**Review by twojtylak — APPROVED** [review]
👍 Cool but I'm afraid of all the conflicts .. 😄

**Comment by michz** [comment]
@twojtylak 
At least don't worry for Morpheus:

> 
>  [ERROR] Found 5 errors.
> 
>  [OK]  ALL of them are fixable! Just add "--fix" to console command and rerun to apply.
>

**Pull request merged**